apps/src/templates/levelSummary/SummaryResponses.jsx

```diff
@@ -115,31 +131,46 @@ const SummaryResponses = ({
           </div>
         )}
 
-        {/* Section dropdown */}
-        {hasSections && (
-          <label className={styles.sectionSelector}>
-            {i18n.responsesForClassSection()}
-            <SectionSelector reloadOnChange={true} />
-          </label>
-        )}
-
-        {/* Correct answer toggle */}
-        {showAnswerToggle && (
-          <div className={styles.toggleContainer}>
-            <ToggleSwitch
-              isToggledOn={showCorrectAnswer}
-              onToggle={() => {
-                setShowCorrectAnswer(!showCorrectAnswer);
-              }}
-              label={i18n.showAnswer()}
-              expands="summary-correct-answer"
+        <div className={styles.headerContainer}>
+          {/* Section dropdown */}
+          {hasSections && (
+            <label className={styles.sectionSelector}>
+              {i18n.responsesForClassSection()}
+              <SectionSelector reloadOnChange={true} />
+            </label>
+          )}
+
+          {/* Correct answer toggle */}
+          {showAnswerToggle && (
+            <div className={styles.toggleContainer}>
+              <ToggleSwitch
+                isToggledOn={showCorrectAnswer}
+                onToggle={() => {
+                  setShowCorrectAnswer(!showCorrectAnswer);
+                }}
+                label={i18n.showAnswer()}
+                expands="summary-correct-answer"
+              />
+            </div>
+          )}
+          {isFreeResponse && DCDO.get('cfu-pin-hide-enabled', false) && (
```

Comment:
optional nit: move under `{/* Free response visualization */}`

Reply:
Could you include a picture when it has both toggles or will showAnswerToggle and isFreeResponse never both be true?

Reply:
As far as I can tell, both should never be true.  The "Answer" to Free Response is always in the "Teachers Only" section.  AND for multiple choice, we never show names.

<img width="769" alt="image" src="https://github.com/user-attachments/assets/ccaa8daa-07ef-49d6-9fb6-66e92aaca58e">